shared/util/container.tsx

```diff
@@ -99,3 +99,4 @@ export {default as makeReducer} from './make-reducer'
 export type ActionHandler<S, A> = _ActionHandler<S, A>
 export {default as useRPC} from './use-rpc'
 export {default as useSafeCallback} from './use-safe-callback'
+export {default as useFocusBlur} from './use-focus-blur'
```

Comment:
it feels a little odd that the place all the util hooks go is `Container` when half of them have nothing to do with containers.

Reply:
(Not that I have an alternate suggestion but maybe we should think about it in #frontend at some point

Reply:
I informally think of container hooks as ones that interact with state / navigation / other app infra. And common adapter ones as purely view things.